dist/yellow-text.js

```diff
@@ -252,6 +247,21 @@
 
       /**
       *
+      * hasContentChanged
+      * =========================================
+      * This function check if the content has changed
```

Comment:
Could you clarify in the comment of the function that this function triggers the `this.options.isContentChanged` callback instead of only checking if the content has changed.